src/audit-logs/serializers/audit-log-export-options.serializer.ts

```diff
@@ -0,0 +1,17 @@
+import {
+  AuditLogExportOptions,
+  SerializedAuditLogExportOptions,
+} from '../interfaces';
+
+export const serializeAuditLogExportOptions = (
```

Comment:
Very happy to move around these serializers and deserializers. Basically, I'm looking to serialize to snake case and everything we get back from the API I'm deserializing back to camelCase. 

Not married to the folder structure, actually just taking a pattern I've seen in Ember before.

Reply:
They seem a bit spread apart right now. I think it could be nice to have a given serializer/deserializer pair in the same file.

Reply:
Centralized them all in the `serializers` folder in https://github.com/workos/workos-node/compare/6cdcc44b85cbd3f57adbf303861454d2ffb89fa1..6d97bbf266d984493a97db8a3b61d5c1456bd3d7